alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -615,7 +620,7 @@ class DocumentLevelMonitorRunner : MonitorRunner() {
         )
     }
 
-    private suspend fun updateLastRunContext(
```

Comment:
Are we sure we do not need suspend anymore? I recall we needed it before

Reply:
we needed to do suspend earlier because we had a search call to calculate max seq_no being made within the method. 
we have removed the search call as we calculate the max seq_no in the first pass. hence no need of a suspend